pkg/controller/plan/adapter/ova/builder.go

```diff
@@ -513,9 +513,10 @@ func getResourceCapacity(capacity int64, units string) (int64, error) {
 	if units == "" {
 		return 0, nil
 	}
-
+	if units == "byte" {
+		return capacity, nil
+	}
```

Comment:
this looks alright but looking at the code below, seems there are more cases that aren't covered - can we make it closer to [the code in vdsm](https://github.com/oVirt/vdsm/blob/1938e3cb68bd658c53b042335fc830414fc238bd/lib/vdsm/v2v.py#L1381-L1430)?

Reply:
or similar to [the code in ovirt-engine](https://github.com/oVirt/ovirt-engine/blob/master/backend/manager/modules/utils/src/main/java/org/ovirt/engine/core/utils/ovf/OvfOvaReader.java#L251-L270)